arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
```

Comment:
Moved this generic description to composite and updated the one here.

---
In reply to: [85594905](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85594905) [](ancestors = 85594905)